### DISCUSSION TITLE
Trouble getting Remote authentication working in a docker with Traefik and Authentik

### DISCUSSION THREAD

**Discussion by pwd-rh** [netbox-community/netbox#16914] [Help Wanted!]
I am new to Netbox and have gotten it working with Traefik and Authentik without problem except I want to use the
Authentik proxy for authentication. 

The env/netbox.env file has the following lines in it.

```
SKIP_SUPERUSER=true
WEBHOOKS_ENABLED=true
REMOTE_AUTH_ENABLED=true
REMOTE_AUTH_HEADER=X-Authentik-Username
LOGLEVEL=DEBUG

```
When I replace the netbox server in my proxy setup with whoami the output contains

```
X-Authentik-Username: pwd
```
pwd is the initial superuser account and if I hit the Log In button I can log in correctly, and the user is set up correctly.

I did restart the docker compose after updating the env file. I am not sure what
I am doing incorrectly.

**Comment by candlerb** [comment]
If the actual header being sent is "X-Authentik-Username: ...", then you need `REMOTE_AUTH_HEADER=HTTP_X_AUTHENTIK_USERNAME`

If that doesn't fix it, then you should at least describe in what way it's not working. (What does the user see when they try to connect?)

**Reply by pwd-rh** [reply]
That was it exactly, I did not know that I needed to put the HTPP_ at the start and make it all caps.

What WAS happening was that when the user went to Netbox they then had to log to netbox and where
not already logged in. It is working now.
